### DISCUSSION TITLE
Flows in a 'suspended' state

### DISCUSSION THREAD

**Discussion by gcusworth** [microsoft/powerapps-tools#978] [Q&A] [answered]
Does anyone know what causes a flow to be in a suspended state on the COE Power BI dashboard, other than violating DLP policies or billing restrictions? For example, does the state change to 'suspended' if there is an error in the flow? Thanks

**Comment by manuelap-msft** [comment, marked as answer]
Only DLP, license or API call limits would cause a flow to be suspended. Flows that run with errors would still show as Started
